services/app-web/src/components/SubmissionSummarySection/UploadedDocumentsTable/UploadedDocumentsTable.module.scss

```diff
@@ -38,3 +38,14 @@
     display: flex;
     justify-content: space-between;
 }
+
+.newDocTag {
+    background-color: #009ec1;
```

Comment:
@haworku I couldn't get `$theme-color-info-dark` to work, even when I uncommented the entry in the master colors file.  Not sure what the problem is there.

Reply:
Here is the new issue https://qmacbis.atlassian.net/browse/MR-2067